configs/openSUSE/dbus-services.toml

```diff
@@ -219,6 +219,20 @@ nodigests = [
     "/usr/share/dbus-1/system.d/org.kde.fontinst.conf",
 ]
 
+[[FileDigestGroup]]
```

Comment:
Commit message for this change does not contain bugzilla reference (bsc#1217186)

Reply:
fixed